.circleci/config.yml

```diff
@@ -23,71 +19,65 @@ parameters:
+  integration-test_using-exe:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - msix/pack:
+          working-dir: sample/SampleExe
+          using-sln: false
+          mapping-file: mapping.txt
+          manifest-file: appxmanifest.xml
+      - msix/sign:
+          working-dir: sample/SampleExe
+          package-name: circleci-cli
+          signing-cert: MSIX_SIGNING_CERT
+          certificate-fingerprint: "91E805BC501F4951E2B48E222DA6C3D882BFA683"
 
 workflows:
-  # Prior to producing a development orb (which requires credentials) basic validation, linting, and even unit testing can be performed.
-  # This workflow will run on every commit
   test-pack:
     unless: << pipeline.parameters.run-integration-tests >>
     jobs:
       - orb-tools/lint # Lint Yaml files
       - orb-tools/pack # Pack orb source
-      - shellcheck/check:
-          dir: ./src/scripts
-          exclude: SC2148
-      # optional: Run BATS tests against your scripts
-      - bats/run:
-          path: ./src/tests
-      # Publish development version(s) of the orb.
       - orb-tools/publish-dev:
           orb-name: circleci/microsoft-msix
-          context: orb-publishing # A restricted context containing your private publishing credentials. Will only execute if approved by an authorized user.
+          context: orb-publishing
           requires:
             - orb-tools/lint
             - orb-tools/pack
-            - bats/run
-            - shellcheck/check
-      # Trigger an integration workflow to test the
-      # dev:${CIRCLE_SHA1:0:7} version of your orb
       - orb-tools/trigger-integration-tests-workflow:
           name: trigger-integration-dev
           context: orb-publishing
           requires:
             - orb-tools/publish-dev
 
-  # This `integration-test_deploy` workflow will only run
-  # when the run-integration-tests pipeline parameter is set to true.
-  # It is meant to be triggered by the "trigger-integration-tests-workflow"
-  # job, and run tests on <your orb>@dev:${CIRCLE_SHA1:0:7}.
   integration-test_deploy:
     when: << pipeline.parameters.run-integration-tests >>
     jobs:
-      # Run any integration tests defined within the `jobs` key.
-      - integration-test-1
-      # Publish a semver version of the orb. relies on
-      # the commit subject containing the text "[semver:patch|minor|major|skip]"
-      # as that will determine whether a patch, minor or major
-      # version will be published or if publishing should
-      # be skipped.
-      # e.g. [semver:patch] will cause a patch version to be published.
+      - integration-test_using-sln
+      - integration-test_using-exe
       - orb-tools/dev-promote-prod-from-commit-subject:
           orb-name: circleci/microsoft-msix
           context: orb-publishing
           add-pr-comment: false
           fail-if-semver-not-indicated: true
           publish-version-tag: false
```

Comment:
Enable tagging
  Add the ssh key with write access and the orb-publisher bot user.
enable PR commenting.
will slack confluence document.